src/main/resources/pdf_templates/ds01.html

```diff
@@ -0,0 +1,229 @@
+
+    <style type="text/css">
+        body {
+            padding: 20px 40px 0 40px;
+            font-size: 14px;
+            font-family: 'TransportNewLight-Regular', sans-serif;
+        }
+
+        img {
+            height: 100%;
+            width: 100%;
+        }
+
+        hr {
+            border: 0.7px solid black;
+        }
+
+        .ch-logo {
+            margin-top: 20px;
+            width: 280px;
+            height: 40px;
+            float: left;
+        }
+
+        .application-type {
+            float: right;
+            text-align: right;
+            margin-right: 10px;
+            font-weight: bold;
+            width: 300px;
+        }
+
+        .application-type-big {
+            font-size: 55px;
+        }
+
+        .application-type-small {
+            font-size: 18px;
+        }
+
+        header {
+            display: block;
+            height: 270px;
+        }
+
+        h1 {
+            width: 350px;
+            float: right;
+            border: 5px solid black;
+            padding: 10px 15px 5px 15px;
+            text-align: center;
+            font-size: 30px;
+            display: block;
+            font-weight: normal;
+            margin-top: 0px;
+            margin-bottom: 5px;
+        }
+
+        h2 {
+            text-align: center;
+            font-weight: bold;
+            font-size: 20px;
+            margin-top: 10px;
+            margin-bottom: 10px;
+        }
+
+        h3 {
+            font-weight: bold;
+            margin-bottom: 5px;
+            margin-top: 10px;
+        }
+
+        p {
+            margin-top: 15px;
+            margin-bottom: 20px;
+            line-height: 20px;
+        }
+
+        .company-data {
+            margin-top: 20px;
+            width: 100%;
+        }
+
+        .company-data-left {
+            width: 175px;
+        }
+
+        .company-data table tr {
+            height: 25px;
+        }
+
+        .director-data-left {
+            padding-left: 10px;
+            width: 210px;
+        }
+
+        .director-data-right {
+            width: 320px;
+            padding-top: 30px;
+        }
+
+        .hr-thin {
+            border: 0.2px solid black;
+            width: 80%;
+            float: left;
+        }
+
+        .hr-div {
+            width: 100%;
+            float: left;
+        }
+
+        .director-data-table tr {
+            height: 100px;
+            page-break-inside: avoid;
+        }
+
+        table {
+            width: 100%;
+        }
+
+        .label {
+            display: block;
+        }
+
+        #bottom-line {
+            margin-top: 15px;
+        }
+
+        .info {
+            width: 100%;
+            height: 125px;
+        }
+    </style>
+</head>
+
+<body xml:lang="en">
+<header class="upper">
+    <div class="info">
+        <div class="ch-logo">
+            <img src="https://document-render-assets-registry.s3-eu-west-1.amazonaws.com/dissolution/images/ch_logo.jpg"
```

Comment:
Wait, we don't want to hardcode this. In production we won't have access to this S3 Bucket